File: src/kyu_8/holiday_VI_shark_pontoon/python/solution.py
def shark(pontoonDistance, sharkDistance, youSpeed, sharkSpeed, dolphin):
    if dolphin:
        sharkSpeed = sharkSpeed * 0.5

    if pontoonDistance / youSpeed < sharkDistance / sharkSpeed:
        return "Alive!"
    else:
        return "Shark Bait!"
    

File: src/kyu_8/holiday_VI_shark_pontoon/js/solution.js
function shark(pontoonDistance, sharkDistance, youSpeed, sharkSpeed, dolphin) {
    const isDolphin = dolphin === true ? sharkSpeed / 2 : sharkSpeed;
    if ((sharkDistance / isDolphin) > (pontoonDistance / youSpeed)) {
        return "Alive!";
    } else {
        return "Shark Bait!";
    }
}

module.exports = shark;